features/step_definitions/show_stats_steps.rb

```diff
@@ -30,7 +30,7 @@
   step "I run `#{command}`"
 
   result = Hash.new(0)
-  in_current_dir do
+  cd('.') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.